_posts/2015-09-20-mikaila-waters-illustrator.markdown

```diff
@@ -0,0 +1,26 @@
+---
+layout:     post
+title:      "Illustrator for Developers and SVGs"
+subtitle:   "Illustrator basics and how to create an SVG icon"
+date:       2015-09-21 12:00:00
+author:     "Mikaila Waters"
+header-img: "img/illustrator.jpg"
+---
+
+## Background on Mikaila
+
+Mikaila is a current student in the Firehose Project curriculum and works at an edtech startup called Gradeable on all aspects of the product. She started at Gradeable as the designer, coming from a background in Architecture, and taught herself HTML, CSS, and Javascript.
+
+Mikaila walks you through the common tools she uses in Illustrator and how to create an SVG image for your front end. Get in touch with Mikaila [here](https://twitter.com/oh_my_kaila).
+
+{% include youtubePlayer.html id="8ZJZ0zjxGxk" %}
+
+
+Watch the full video [here](https://youtu.be/8ZJZ0zjxGxk).
```

Comment:
That would be a good idea but I don't think markdown allows you to write inline comments like we would with Ruby/Rails, etc. However, what we could do is either change the wording of the sentence to "Want to see more than just this video? Watch the full Firehose Project lightning talks here" (something along those lines) or we could add it to the contributors documentation. 

I like that idea though since I think there was only 1 person who didn't confuse them at the PR because they asked before they submitted.